docs/source/gbd2017_models/causes/neoplasms/colon_and_rectum_cancer/cancer_model.rst

```diff
@@ -1,244 +1,274 @@
-   
-   * - Measure
-     - Sources
-     - Description
-     - Notes
-   * - prevalence_c441
-     - como
-     - Prevalence of cause colon and rectum cancer
-     - 
-   * - deaths_c441
-     - codcorrect
-     - Deaths from colon and rectum cancer
-     - 
-   * - population
-     - demography
-     - Mid-year population for given country
-     - 
-   * - incidence_rate_c441
-     - como
-     - Incidence rate for colon and rectum cancer
-     - 
-   * - disability_weight_s{`sid`}
-     - YLD appendix
-     - Disability weights associated with each sequelae
-     - 
-   * - prevalence_s{`sid`}
-     - como
-     - Prevalence of each sequelae
-     - 
-
-
-Validation Criteria
-+++++++++++++++++++
-
-.. todo::
-
-   Describe tests for model validation.
-
-
-References
-----------
-
-.. [GBD-2017-YLD-Capstone-Appendix-1-Colon-and-rectum-Cancer]
-   Supplement to: `GBD 2017 Disease and Injury Incidence and Prevalence
-   Collaborators. Global, regional, and national incidence, prevalence, and
-   years lived with disability for 354 diseases and injuries for 195 countries
-   and territories, 1990–2017: a systematic analysis for the Global Burden of
-   Disease Study 2017. Lancet 2018; 392: 1789–858`
-   (pp. 310-317)
+.. _2017_cancer_model_colon_and_rectum_cancer:
+
+=======================
+Colon and Rectum Cancer
+=======================
+
+.. contents::
+  :local:
+
+Disease Overview
+++++++++++++++++
+
+.. todo::
+
+ add CRC overview
+
+.. list-table:: Malignant neoplasm of colon and rectum cancer 
+   :widths: 10 10
+   :header-rows: 1
+
+   * - ICD 10
+     - ICD 9
+   * - C18-C19.0, C20, C21-C21.8, Z12.1-Z12.13 (Z*=screening)
+     - 153-154.9, V76.41, V76.5-V76.52 (V*=screening)
+
+
+GBD 2017 Modeling Strategy
+++++++++++++++++++++++++++
+
+.. todo::
+
+ add GBD CRC modeling strategy
+
+
+Cause Hierarchy
+------------------------------------------
+       
+.. image:: colon_and_rectum_cancer_cause_hierarchy.svg
+
+
+Restrictions
+------------
+
+The following table describes any restrictions on the effects of this cause 
+(such as being only fatal or only nonfatal), as well as restrictions on the 
+age and sex of simulants to which different aspects of the cause model applies.
+
+.. list-table:: Restrictions
+   :widths: 10 10 10
+   :header-rows: 1
+
+   * - Restriction Type
+     - Value
+     - Notes
+   * - Male only
+     - False
+     -
+   * - Female only
+     - False
+     -
+   * - YLL only
+     - False
+     -
+   * - YLD only
+     - False
+     -
+   * - YLL age group start
+     - 15 to 19
+     - GBD age group id 8
+   * - YLL age group end
+     - 95 plus
+     - GBD age group id 235
+   * - YLD age group start
+     - 15 to 19
+     - GBD age group id 8
+   * - YLD age group end
+     - 95 plus
+     - GBD age group id 235
+
+
+Vivarium Modeling Strategy
+++++++++++++++++++++++++++
+
+Things to consider: 
+
+1. Duration of remission from clinical CRC *over 10 years*, to be confirmed 
+   with GBD modeler.
+

```

Comment:
My understanding is that conceptually remission of the *cancer* is still 10 years, but that disability due to the stoma can extend beyond 10 years, right? Maybe we will have to remove that sequela from the prevalence of the CRC cause so that it is compatible with what we want it to represent?

Reply:
Good catch, include this sequelae can overestimate YLD.